server/utils/nunjucks.summaryFields.ts

```diff
@@ -172,11 +164,9 @@ export function resolveNextStep(next: FormWizard.Step.NextStep, ctx: Context): F
     throw new Error(`unable to resolve ${con.next} - callbacks are not supported yet`)
   }
 
-  for (const nestedNext of next as FormWizard.Step.NextStep[]) {
-    const check = resolveNextStep(nestedNext, ctx)
-    if (check !== undefined) {
-      return check
-    }
+  if (Array.isArray(next)) {
+    const resolvable = next.find(nestedNext => resolveNextStep(nestedNext, ctx) !== undefined)
+    return resolveNextStep(resolvable, ctx)
```

Comment:
@LucasCairns is there some clever way to do this without a) calling `resolveNextStep` twice; and b) iterating over the entire `next` array? 🤔